index.bs

```diff
@@ -2421,6 +2415,106 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 the same procedure as other [=assertion signatures=] generated by the [=authenticatorMakeCredential=] operation.
 
 
+## Authenticator taxonomy ## {#sctn-authenticator-taxonomy}
+
+Different kinds of [=authenticators=] enable different use cases along two independent dimensions: the [=authentication ceremony
+structure=] and the [=authenticator=]'s [=attachment modality=]. The [figure below](#fig-authData) illustrates how the use cases
+relate to different kinds of [=authenticators=].
+
+<figure id="fig-authenticator-taxonomy">
+    <img src="images/authenticator-taxonomy.svg"/>
+    <figcaption>Relationships between [=authenticator=] properties and use cases.</figcaption>
+</figure>
+
+The terms [=second-factor use case|second-factor=], [=first-factor use case|first-factor=], [=single-step use case|single-step=]
+and [=username-less use case|username-less=] referenced in the figure refer to the use cases defined in
+[[#sctn-authentication-ceremony-structures]]. The terms [=platform authenticator|platform=] and [=roaming authenticator|roaming=]
+are defined in [[#sctn-authenticator-attachment-modality]]. The terms [=user verification=] and [=client-side-resident credential
+private key=] are defined in [[#terminology]].
+
+
+### <dfn>Authentication ceremony structures</dfn> ### {#sctn-authentication-ceremony-structures}
+
+The [=authentication ceremony structure=] is defined by the [=authentication factors=] used in the [=authentication
+ceremony|ceremony=]. Different [=authentication ceremony structures=] provide different benefits for security and usability.
+
```

Comment:
Plus, there are even higher-level use cases given RPs' and users' various relationship lifecycle phases. See the musings in https://github.com/w3c/webauthn/issues/334#issue-203817746 and https://github.com/w3c/webauthn/issues/334#issuecomment-388989355.

Those two musings intersect but are not zactly congruent, and also intersect with the use cases described below. 

they most all implicitly assume that the user has an existing account at the RP, which I find confusing. 

I'm thinking we'll need to come to common agreement on use cases and how we describe them (all three (the below and the above two) description approaches are somewhat different).  I'm also trying to coalesce something from all three but have not completed it as yet.

Reply:
I don't know what to do here. I agree there are some things in https://github.com/w3c/webauthn/issues/334#issue-203817746 and https://github.com/w3c/webauthn/issues/334#issuecomment-388989355 that could be nice to incorporate, but I don't really know how to go about it. Shall I wait for you to finish a draft of it?

>they most all implicitly assume that the user has an existing account at the RP, which I find confusing because it is not explicitly called out as a pre-condition.

Well, this particular section is "_Authentication_ ceremony structures", so yes, it's oriented around the authentication operation. "Authenticating" without an account (or, more generally, something to claim) doesn't really make sense, does it? :)